release/models/platform/openconfig-platform-transceiver.yang

```diff
@@ -451,9 +457,11 @@ revision "2023-08-30" {
 
     container thresholds {
       description
-        "Enclosing container for transceiver alarm thresholds.
-        Each threshold is compared to the instant value of the
-        measured parameter corresponding to the threshold";
+        "Enclosing container for transceiver alarm thresholds. Each threshold
+        is compared to the instant value of the measured parameter
+        corresponding to the threshold. For optical input power, the
+        threshold applies only to the physical-channel(s).  This threshold
+        does not apply to the aggregate transceiver input-power.";
```

Comment:
Should we also clarify that the threshold defined applies to every physical channels?

Reply:
Done